### ISSUE TITLE
Test: `accessibilityHelpContent` view property API

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210668]
Refs #209855

- [x] Windows @dbaeumer 
- [x] macOS @paulacamargo25 
- [x] Linux @rzhao271 

Complexity: 3

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23210668%0A%0A&assignees=meganrogge)

---

Accessibility help dialogs exist around the workbench - in the terminal, chat input, editor, and more. With focus in these areas, in screen reader mode, you can view them by pressing `alt+f1`. 

Extensions, like the GHPR extension, want to contribute help dialogs. There is proposed API to allow adding this to a view via `accessibilityHelpContent`. 

Add this for a view. When your focus is in the view, use `alt+f1` and you should see the content that you've provided. 

Remember to include: `"contribAccessibilityHelpContent"` in your `"enabledApiProposals"`. Verify that it does not work if you don't include this.

https://github.com/microsoft/vscode/blob/7b49653d3a5e47284e72b6a858ed1a90cda25141/src/vs/workbench/api/browser/viewsExtensionPoint.ts#L174

Verify that when you provide keybindings in the format of `<keybinding:commandId>`, it's resolved or there's text indicating there's no keybinding with a link to configure. Verify the link to configure it brings you to the keybindings editor with that command id.

Here is an example of what it will look like:
![322630232-5bed81a6-c411-4fb8-a02e-3966667e9ee6](https://github.com/microsoft/vscode/assets/29464607/0a5731cd-1fdb-4d15-af19-43a67ac60505)

**Comment by paulacamargo25** [comment]
@meganrogge i tried to add a debug view but it didnt work, do you have an example on how to add "accessibilityHelpContent"? Thanks
